worlds/AutoWorld.py

```diff
@@ -148,7 +150,10 @@ class World(metaclass=AutoWorldRegister):
     # increment this every time something in your world's names/id mappings changes.
     # While this is set to 0 in *any* AutoWorld, the entire DataPackage is considered in testing mode and will be
     # retrieved by clients on every connection.
-    data_version: ClassVar[int] = 1
+    #
+    # Deprecated. This is only used by older clients. Newer clients should be checking the checksum in DataPackage
+    # instead.
+    data_version: ClassVar[int] = 0
```

Comment:
0 currently still implies to embed the relevant game's datapackage into the resulting .archipelago file.
This PR should probably explain this or offer an alternative, as with this default and explanation I'd expect people to use version 0 now, as in not change it, which is undesired.

Reply:
Hopefully the changes to the comment should be more clear.